plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/render/displayAdapter/impl/EventHandler.java

```diff
@@ -60,79 +60,73 @@ public void handleEvent( Event event ) {
             break;
         case SWT.Resize:
             controlResized(event);
+            break;
         case SWT.KeyDown:
-        // System.out.println("keydown");
+            break;
```

Comment:
I guess since there is no action for key-down, this case-block for `SWT.KeyDown` should be removed